4-hangman/hangman02.py

```diff
@@ -0,0 +1,13 @@
+import random
+my_list = ["I", "eat", "chicken", "and", "turkey"]
+random_word = str(random.choice(my_list))
+# print(random_word.upper())
+
+word = input("Kindly type one letter only: ")
+index = 0
+while index < len(random_word):
```

Comment:
Great job. Just for educational purposes, try this out with a for loop. That way, you don't have to keep track of an index

Reply:
Thanks
Should i edit it?

Reply:
Yeah, for loop works better here